server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -120,10 +120,12 @@ def s3_entry_list_creation(self):
             while True:
                 resp = self.s3_config_obj.list_objects(**kwargs)
                 for obj in resp['Contents']:
-                    md5_returned = obj['ETag'].strip("\"")
+                    md5_returned = self.s3_config_obj.header_md5(obj)
                     self.content_list.append(Entry(obj['Key'], md5_returned))
+                self.logger.debug('list_objects: got {} objects', len(resp['Contents']))
                 try:
                     kwargs['ContinuationToken'] = resp['NextContinuationToken']
+                    self.logger.debug('list_objects: continuing...')
```

Comment:
If we use a variable to exit the loop, then we can calculate the that variable and use that variable to collapse the debug statement at line 128 and 125 so that we don't emit two of them.

Reply:
OK - thanks.